### ISSUE TITLE
Normalize LCH similarities

### ISSUE THREAD

**Issue by lgazpio** [lgazpio/PhrasIS-baselines#13]
Max value is 3.63758615972639

**Comment by lgazpio** [comment]
Incorporate a DataFrame normalizer function in Utils, to normalize all columns of features with domain != [ -1, 1]

**Comment by lgazpio** [comment]
https://scikit-learn.org/stable/modules/generated/sklearn.preprocessing.StandardScaler.html

https://github.com/lgazpio/UD_Intelligent_Systems/blob/master/7%20-%20Machine%20Learning/K-nn/KNN%20people%20categories.py

1 -> identifi left_strip_tokenized_len, right_strip_tokenized_len, left-right, right-left, 2 x lch, 4 x jcn, root=true, |chunk1-chunk2| ,  minimum_difference, maximum_difference

2 -> lista bat egin hauekin features = ["", "" , ""]

3 -> extraitu gure datasetetik feature hauek dataset [ lista ]

X = data.drop("custcat", axis=1, inplace=False).values  ?
X = preprocessing.StandardScaler().fit_transform(X.astype(float))

4 -> zutabe zaharrak zanpatu berrikin
